src/igl/opengl/PlatformDevice.cpp

```diff
@@ -93,10 +93,10 @@ void PlatformDevice::blitFramebuffer(const std::shared_ptr<IFramebuffer>& src,
           "the other doesn't.\n");
     }
     if (srcDepthTexture && dstDepthTexture) {
-      auto srcFormat = std::static_pointer_cast<igl::opengl::Texture>(srcDepthTexture)
-                           ->getGLInternalTextureFormat();
-      auto dstFormat = std::static_pointer_cast<igl::opengl::Texture>(dstDepthTexture)
-                           ->getGLInternalTextureFormat();
+      auto srcFormat = static_cast<igl::opengl::Texture&>(*srcDepthTexture)
```

Comment:
Instead of dereferencing and casting to a reference, please use `.get()` and cast to a raw pointer.

Reply:
done